src/Symfony/Component/HttpClient/Tests/TraceableHttpClientTest.php

```diff
@@ -80,4 +82,19 @@ public function testItResetsTraces()
         $sut->reset();
         $this->assertCount(0, $sut->getTracedRequests());
     }
+
+    public function testStream(): void
```

Comment:
please remove `void` - test cases don't need that kind of "guarantees", better reduce the visual debt

Reply:
Yes ! Sorry I put it mechanically